error TS2345: Argument of type 'string | null' is not assignable to parameter of type

Doesn't work with Node.js 14.8

```
src/index.ts:49:40 - error TS2345: Argument of type 'string | null' is not assignable to parameter of type 'string | Uint8Array | Uint8ClampedArray | Uint16Array | Uint32Array | Int8Array | Int16Array | In...'.
Type 'null' is not assignable to type 'string | Uint8Array | Uint8ClampedArray | Uint16Array | Uint32Array | Int8Array | Int16Array | In...'.

49             fs.writeFileSync(filePath, content)
```

This seems to fix it
```diff
diff --git a/src/index.ts b/src/index.ts
index e3c3e67..f274fa5 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -46,7 +46,7 @@ try {
const content = consumer.sourceContentFor(source)
const filePath = `${process.cwd()}/${projectNameInput}/${source.substring(WEBPACK_SUBSTRING_INDEX)}`
mkdirp.sync(dirname(filePath))
-            fs.writeFileSync(filePath, content)
+            fs.writeFileSync(filePath, content as string)
})
console.log(chalk.green('🎉  All done! Enjoy exploring your code 💻'))
})
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -43,7 +43,7 @@ try {
         const sources = (consumer as any).sources
         sources.forEach((source: string) => {
             const WEBPACK_SUBSTRING_INDEX = 11
-            const content = consumer.sourceContentFor(source)
+            const content = consumer.sourceContentFor(source) as string
             const filePath = `${process.cwd()}/${projectNameInput}/${source.substring(WEBPACK_SUBSTRING_INDEX)}`
             mkdirp.sync(dirname(filePath))
             fs.writeFileSync(filePath, content)

